### ISSUE TITLE
Mark themes which don't require external Webfonts

### ISSUE THREAD

**Issue by kaihendry** [KyleAMathews/typography.js#79]
Sorry, I raised the issue before by email or twitter. However my wish remains the same. A method to see which themes do not use external fonts. Currently there is no easy way to tell upon:
http://kyleamathews.github.io/typography.js/

My end goal is to evaluate themes (or font styles) that do not bring in external fonts. External fonts are a no go in terms of Web performance for me and I'm trying to get the best "look" between major platforms like Android & IOS.

**Comment by KyleAMathews** [comment]
I agree this is a very desirable feature. This is on the roadmap for the next iterations of the design tools.

In the meantime, these are the themes that don't use any external fonts:

* default
* bootstrap
* github
* wikipedia
* wordpress 2010
* wordpress 2011
* wordpress kubrick

You can also create your own theme w/ system fonts of your own choice. Here's a website with a good listing of different "font stacks" http://www.cssfontstack.com/

All options for the API are optional. So start with `const typography = new Typography({})` and then start adding options e.g. changing the `bodyFontFamily` and `headerFontFamily` — https://github.com/KyleAMathews/typography.js#api

/cc @elesant
